### PR TITLE
Publish swagger docs

### PR DESCRIPTION
**Pull request by vlaurin**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/RDM-2902

### Change description ###

Add config to publish swagger docs automatically from Travis

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -20,5 +20,9 @@ env:
   global:
     - CODACY_PROJECT_TOKEN=24f7bf62b3fa4328aa3b9bf32c07b0d9
 
+before_install:
+  - curl https://raw.githubusercontent.com/hmcts/reform-api-docs/master/bin/publish-swagger-docs.sh > publish-swagger-docs.sh
+
 after_success:
   - "bash <(curl -s https://codecov.io/bash) || echo 'Codecov failed to upload'"
+  - test "$TRAVIS_BRANCH" = "master" && test "$TRAVIS_PULL_REQUEST" = "false" && sh ./publish-swagger-docs.sh
```
Comment (dwaynebailey) [inline .travis.yml]:
```suggestion
  - [ "$TRAVIS_BRANCH" = "master" -a "$TRAVIS_PULL_REQUEST" = "false" ] && sh ./publish-swagger-docs.sh
```

Comment (dwaynebailey) [reply .travis.yml]: I think the above will work and it's a little cleaner, not critical.

File: .env [hunk 2]
```diff
@@ -1,3 +1,4 @@
 # Docker environment
 DATA_STORE_DB_USERNAME=ccd
 DATA_STORE_DB_PASSWORD=ccd
+SERVER_PORT=4452
```
Comment (dwaynebailey) [inline .env]: I assume this cleanup isn't directly needed for the swagger docs? `SERVER_PORT` does make me wonder if we can use this in Dockerfiles... he thinks aloud.

Comment (vlaurin) [reply .env]: it's needed in the way that the publish script needs a `SERVER_PORT` env defined and achieves that by sourcing the `.env` file.

**Pull request merged**